packages/sonarwhal/src/lib/connectors/utils/requester.ts

```diff
@@ -177,6 +177,72 @@ export class Requester {
         return this._redirects.calculate(uri);
     }
 
+    /**
+     * Performs a `HEAD` request to the given `uri`
+     */
+    public head(uri: string): Promise<NetworkData> {
+        debug(`Requesting(HEAD) ${uri}`);
+
+        return new Promise((resolve: Function, reject: Function) => {
+            let rawBodyResponse: Buffer;
+
+            request.head({ uri }, async (err, response) => {
+                if (err) {
+                    debug(`Request(HEAD) for ${uri} failed\n${err}`);
+
+                    return reject({
+                        error: err,
+                        uri
+                    });
+                }
+                // We check if we need to redirect and call ourselves again with the new target
+                if (Requester.validRedirects.includes(response.statusCode)) {
+                    const newUri = url.resolve(uri, response.headers.location[0]);
```

Comment:
Why are you doing here `[0]`? We are not doing it in `get`. If it's an error there we should fix it.